src/unlocked/didcomm.rs

```diff
@@ -0,0 +1,162 @@
+extern crate didcomm_rs;
+
+use std::convert::TryInto;
+use crypto_box::PublicKey;
+use didcomm_rs::{DidcommHeader, Jwk, KeyAlgorithm, Message, crypto::{SignatureAlgorithm, Signer}};
+use x25519_dalek::{StaticSecret};
+use crate::{prelude::*, Error, unlocked::UnlockedWallet, contents::Content};
+
+impl UnlockedWallet {
+    /// Helper function to get JSON String of default empty `Message`
+    /// 
+    pub fn create_message() -> String {
```

Comment:
why have this method?

Reply:
this allows to have JS structure of a blank message as json with the didcomm default struct. if this will change on rust level - this method will yield new json matching that. no need to maintain structure on JS side.